[word] (Requirement Sets) Splitting Word requirement set pages (#1185)
OfficeDev/office-js-docs-pr: docs/reference/requirement-sets/word-api-1-1-requirement-set.md

---
title: Word JavaScript API requirement set 1.1
description: 'Details about the WordApi 1.1 requirement set'
ms.date: 07/17/2019
ms.prod: word
localization_priority: Normal
---

# What's new in Word JavaScript API 1.1

> Word JavaScript API 1.1 is the first version of the API. It is the only Word-specific requirement set supported by Excel 2016.

Reviewer: In the other requirement set articles, you refer to the set using this convention: "WordApi 1.3".  For consistency, should "Word JavaScript API 1.1." in this sentence be changed to -> "WordApi 1.1" ?  The suggestion I've made below incorporates this change and a couple of other wording changes as well (including "Excel" -> "Word").

Reviewer: ```suggestion
WordApi 1.1 is the first requirement set of the Word JavaScript API. It's the only Word API requirement set supported by Word 2016.
```

> ## API List

Reviewer: ```suggestion
## API list
```

> The following is a complete list of APIs added as part of the WordApi 1.1 requirement set.

Reviewer: ```suggestion
The following table lists the APIs added as part of the WordApi 1.1 requirement set.
```
